src/main/java/com/google/javascript/clutz/DeclarationGenerator.java

```diff
@@ -68,6 +68,8 @@
  */
 public class DeclarationGenerator {
 
+  public static final String INSTANCE_CLASS_SUFFIX = "_Instance";
```

Comment:
I presume we won't run into collisions with actual user code with this, as `_` violates Closure style, but might be something we have to keep in mind for the one obscure case that always comes up.